docs/modules/ROOT/pages/index.adoc

```diff
@@ -826,9 +826,9 @@ IMPORTANT: The trust decisions described in the scenarios should only be evaluat
 
 The trust decision can result in one of three descriptions of the _<<_named_actor,named actor>>:_
 
-* *Trusted:* The _<<_identity_asertion_consumer,identity assertion consumer>>_ can verify a direct or indirect trust relationship to the _<<_named_actor,named actor>>_ through its established roots of trust.
-* *Well-formed:* The _<<_identity_asertion_consumer,identity assertion consumer>>_ can not verify a direct or indirect trust relationship to the _<<_named_actor,named actor>>._ 
-* *Revoked:* The credential used for signing the *<<_identity_assertion,identity assertion>>* has been revoked by the credential’s issuer.
+* *Trusted:* The _<<_identity_asertion_consumer,identity assertion consumer>>_ can verify a direct or indirect trust relationship to the _<<_named_actor,named actor>>_ through its established roots of trust. The _<<_identity_asertion_consumer,identity assertion consumer>>_ found no evidence that the credential was revoked by the credential’s issuer at the timethe *<<_identity_assertion,identity assertion>>* was issued.
+* *Well-formed:* The _<<_identity_asertion_consumer,identity assertion consumer>>_ can not verify a direct or indirect trust relationship to the _<<_named_actor,named actor>>._ The _<<_identity_asertion_consumer,identity assertion consumer>>_ found no evidence that the credential was revoked by the credential’s issuer at the timethe *<<_identity_assertion,identity assertion>>* was issued.
+* *Revoked:* The credential used for signing the *<<_identity_assertion,identity assertion>>* has been revoked by the credential’s issuer at the timethe *<<_identity_assertion,identity assertion>>* was issued.
```

Comment:
I think this should be more explicit...
"* *Revoked:* The credential used for signing the *<<_identity_assertion,identity assertion>>* had been revoked by the credential’s issuer prior to when it was used to sign the *<<_identity_assertion,identity assertion>>*."

Reply:
See my comment in #138. I have the same concern here.